static/gsApp/hooks/genAiAccess.tsx

```diff
@@ -33,7 +30,7 @@ export function useGenAiConsentButtonAccess({
 
   const conditions = [
     {
-      check: !regionData?.name || !['us', 'sentry4sentry'].includes(regionData.name),
+      check: !organization.features.includes('gen-ai-consent'),
```

Comment:
i'm just confused by this new flag and it being called "consent"

Reply:
am i correct in understanding it's temporary just for testing/debugging?

Reply:
it was supposed to be permanent refactor for the consent check to not be region dependant, i didn't realize that we have already started work on removing it.